src/GuzzleHttp/Command/Guzzle/GuzzleClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Here is the documentation to register an event subscriber:

https://github.com/csarrazi/CsaGuzzleBundle/blob/master/src/Resources/doc/event_subscribers.md

Then, you simply need to define a service for your subscriber:

```xml
<service id="csa_guzzle.subscriber.serializer" class="Csa\Bundle\GuzzleBundle\GuzzleHttp\Subscriber\SerializerSubscriber">
    <tag name="csa_guzzle.subscriber" alias="serializer" />
    <argument />
</service>
```

Then, in the DI extension, you simply need to replace the first argument of the `csa_guzzle.subscriber.serializer` service with a reference to the correct serialization service.

Of course, if the serializer is disabled (which should be the default), you should run `$container->removeDefinition()`, as in https://github.com/csarrazi/CsaGuzzleBundle/blob/master/src/DependencyInjection/CsaGuzzleExtension.php#L48, to make sure that we don't enable the subscriber when we don't need it.